metrics/web/test/base/presentation/widgets/scorecard_test.dart

```diff
@@ -1,57 +1,27 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:metrics/dashboard/presentation/widgets/text_metric.dart';
+import 'package:metrics/base/presentation/widgets/scorecard.dart';
 
 import '../../../test_utils/metrics_themed_testbed.dart';
 
 void main() {
   group("TextMetric", () {
-    testWidgets(
-      "can't be created with null description",
-      (WidgetTester tester) async {
-        await tester.pumpWidget(const _TextMetricTestbed(
-          description: null,
-          value: 'value',
-        ));
-
-        expect(
-          tester.takeException(),
-          isAssertionError,
-        );
-      },
-    );
-
-    testWidgets(
-      "can't be created with null value",
-      (WidgetTester tester) async {
```

Comment:
Maybe we leave those tests - but just repurpose to have a predicted and defined behavior when those parameters have null value - should it display empty text then? or null?